database/sql/src/main/scala/cromwell/database/sql/tables/CallCachingJobDetritusEntry.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.database.sql.tables
+
+case class CallCachingJobDetritusEntry
```

Comment:
All case classes should be final. It's been said that if it weren't a backward compat nightmare that they'd go back and make that an official rule